WordPress/src/main/res/values/available_languages.xml

```diff
@@ -26,36 +26,32 @@
     <item>zh_CN</item>
     <item>zh_TW</item>
     <item>az</item>
+    <item>bg</item>
+    <item>cs</item>
+    <item>cy</item>
+    <item>da</item>
     <item>el</item>
-    <item>es_MX</item>
+    <item>en_AU</item>
+    <item>en_CA</item>
     <item>es_CL</item>
+    <item>es_CO</item>
+    <item>es_MX</item>
+    <item>es_VE</item>
+    <item>eu</item>
     <item>gd</item>
+    <item>gl</item>
     <item>hi</item>
+    <item>hr</item>
     <item>hu</item>
+    <item>is</item>
+    <item>kmr</item>
+    <item>mk</item>
+    <item>ms</item>
     <item>nb</item>
-    <item>pl</item>
-    <item>th</item>
-    <item>uz</item>
-    <item>zh_HK</item>
-    <item>eu</item>
     <item>ro</item>
-    <item>mk</item>
-    <item>en_AU</item>
-    <item>sr</item>
     <item>sk</item>
-    <item>cy</item>
-    <item>da</item>
-    <item>bg</item>
     <item>sq</item>
-    <item>hr</item>
-    <item>cs</item>
-    <item>pt_BR</item>
-    <item>en_CA</item>
-    <item>ms</item>
-    <item>es_VE</item>
-    <item>gl</item>
-    <item>is</item>
-    <item>es_CO</item>
-    <item>kmr</item>
+    <item>uz</item>
+    <item>zh_HK</item>
```

Comment:
This change happened in that file because I reordered the list of locales in the `Fastfile` script as part of https://github.com/wordpress-mobile/WordPress-Android/pull/16718 to make them listed alphabetically and remove duplicates.

One should still double-check that this diff makes sense and that the changes in that XML still contain the right list of locales (just in a different order), as this XML that our tooling generates/updates [is then used in the app code to present the in-app locale switcher](https://github.com/wordpress-mobile/WordPress-Android/blob/471f1f5ccbff83f4301d1c5988597a05ce6a468b/WordPress/src/main/java/org/wordpress/android/ui/prefs/language/LocalePickerViewModel.kt#L147-L149)